sway/commands.c

```diff
@@ -213,9 +210,16 @@ list_t *execute_command(char *_exec, struct sway_seat *seat,
 		}
 	}
 
+	char *exec = strdup(_exec);
+	char *head = exec;
+	list_t *res_list = create_list();
+
+	if (!sway_assert(res_list && exec, "allocation failure")) {
```

Comment:
This shouldn't be an assertion. Allocation failures shouldn't make the process abort in debug mode.